packages/core/mixins/_helpers.scss

```diff
@@ -121,3 +121,18 @@
         outline: 0;
     }
 }
+
+///
+/// Removes increment and decrement buttons
+/// for number input fields.
+///
+@mixin remove-inner-outer-button {
```

Comment:
Kanskje den burde heite remove-number-input-spinner? Dette kjem frå nokon som aldri har vore borti input spinneren før, noko som både er ein fordel og ulempe for å gi navn på ting.

Reply:
Enig! Det var først med denne kommentaren jeg forsto hva det var snakk om å fjerne 😅